CHANGELOG.md

```diff
@@ -1,5 +1,77 @@
 # Changelog
 
+## 2.0.0 - [#24](https://github.com/openfisca/country-template/pull/24)
+
+* Technical change
```

Comment:
`Breaking change`?
Besides, on the user side, `HousingOccupancyStatus` type seems to be the main change.
So, what about adding a reference to the documentation on enumerations and remove generic information on enumerations in the changelog?
> Remove details but keep web api, yaml and python examples?
> For python example, use `housing_occupancy_status` and remove text on `name` and `value`?

Reply:
Users of the API will not see HousingOccupancyStatus.
The breaking change is, IMHO, in the name/value paradigm.